cost/turbonomics/rightsize_databases_recommendations/aws/turbonomics_rightsize_databases_recommendations.pt

```diff
@@ -311,6 +470,93 @@ policy "pol_turbonomic_rightsize_databases" do
       field "createdTime" do
         label "Created Time"
       end
+      field "vmAge" do
+        label "VM Age"
+      end
+      field "pricingModel" do
+        label "Pricing Model"
+      end
+      field "url" do
+        label "System Details URL"
+      end
+      field "sizenow" do
+        label "Current GB Size"
+      end
+      field "sizeutil" do
+        label "Current GB Size % Utilization"
+      end
+      field "size" do
+        label "After GB Size"
```

Comment:
Usually, "current size" should be the one named "size", and after size should have another name. Also, label should be something like "After Size (GB)" or "Recommended Size (GB)"